index.bs

```diff
@@ -2352,6 +2356,16 @@ To <dfn>compute the channel capacity of a source</dfn> given a [=randomized resp
 
 Note: This algorithm computes the channel capacity [[CHAN]] of a q-ary symmetric channel [[Q-SC]].
 
+To <dfn>compute the channel capacity of a source with attribution scopes</dfn> given a positive integer |numTriggerStates|, a positive integer |attributionScopeLimit| and a positive integer |maxEventStates|:
+1. Let |totalStates| be |numTriggerStates| + (|attributionScopeLimit| - 1) * |maxEventStates|.
+1. Return log2(|totalStates|).
```

Comment:
Technically, because we disallow a mix of sources with and without attribution scopes, isn't attribution scope limit of 1 the same behaviour as no scopes? If so, why would the limit be different. Can I now game the info check limit by setting the limit to 1?